pkg/pillar/types/attesttypes.go

```diff
@@ -21,29 +17,13 @@ func (nonce AttestNonce) Key() string {
 //SigAlg denotes the Signature algorithm in use e.g. ECDSA, RSASSA
 type SigAlg uint8
 
-//CertType carries the certificate use case e.g. ek, ecdh_exchange etc
-type CertType uint8
-
-//CertHashType carries the hash algo used for compute the short hash
-type CertHashType uint8
-
```

Comment:
Why are you removing this?
Note that we avoid using the enum values from the device internally in EVE and instead have implementation enums defined, to provide isolation between changes in EVE and in the API. So all of this should be unchanged AFAICT.